Work/work.py

```diff
@@ -0,0 +1,85 @@
+print( "WELCOME TO THE PHONEBOOK DIRECTORY") 
+ 
+
+filename = "myphonebook.txt" 
+myfile = open(filename, "a+") 
+myfile.close 
+ 
+ 
+def main_menu(): 
+    print( "\nMAIN MENU\n") 
+    print( "1. Show all existing Contacts") 
+    print( "2. Add a new Contact") 
+    print( "3. Search the existing Contact") 
+    print( "4. Exit") 
+    choice = input("Enter your choice: ") 
+    if choice == "1": 
+        myfile = open(filename, "r+") 
+        filecontents = myfile.read() 
+        if len(filecontents) == 0: 
+            print( "There is no contact in the phonebook.") 
+        else: 
+            print(filecontents) 
+        myfile.close 
+        enter = input("Press Enter to continue ...") 
```

Comment:
Зачем помещать возврат функции `inpout` в переменную? Если значение этой переменной никак в будущем использоваться не будет?
```suggestion
        input("Press Enter to continue ...") 
```